hedera-node/src/main/java/com/hedera/services/grpc/marshalling/AliasResolver.java

```diff
@@ -106,16 +113,19 @@ private List<TokenTransferList> resolveTokenAdjusts(
         final List<TokenTransferList> resolvedTokenAdjusts = new ArrayList<>();
         for (var tokenAdjust : opTokenAdjusts) {
             final var resolvedTokenAdjust = TokenTransferList.newBuilder();
+            tokenTransferResolutions.clear();
 
             resolvedTokenAdjust.setToken(tokenAdjust.getToken());
             for (final var adjust : tokenAdjust.getTransfersList()) {
                 final var result =
                         resolveInternalFungible(
-                                aliasManager, adjust, resolvedTokenAdjust::addTransfers);
-                if (result != Result.KNOWN_ALIAS) {
-                    perceivedMissing++;
-                }
+                                aliasManager, adjust, resolvedTokenAdjust::addTransfers, true);
+
+                // Since the receiver can be an unknown alias in a CryptoTransfer perceive the
+                // result
+                perceiveResult(result, adjust);
```

Comment:
For consistency we might rename this `perceiveNonNftReceiverResult()`; since there's significant duplication with the `perceiveNftReceiverResult()` version; I think it's worth extracting a method,
```
    private void perceiveReceiverResult(
            final Result result,
            final long assetChange,
            final ByteString alias,
            final boolean repetitionsAreInvalid
    ) {
        if (result == Result.UNKNOWN_ALIAS) {
            if (assetChange > 0) {
                if (isSerializedProtoKey(alias)) {
                    perceivedCreations++;
                } else {
                    perceivedInvalidCreations++;
                }
            } else {
                perceivedMissing++;
            }
        } else if (repetitionsAreInvalid && result == Result.REPEATED_UNKNOWN_ALIAS) {
            perceivedInvalidCreations++;
        } else if (result == Result.UNKNOWN_EVM_ADDRESS) {
            perceivedMissing++;
        }
    }
```
so then the implementation of e.g. `perceiveNftReceiverResult()` becomes,
```
        perceiveReceiverResult(
                result, change.getSerialNumber(), change.getReceiverAccountID().getAlias(), false);
```

Reply:
Agreed . Fixed !